### DISCUSSION TITLE
[Backend] Fastify + Typescript

### DISCUSSION THREAD

**Discussion by NullDivision** [realworld-apps/realworld#916] [WIP Implementations]
Hello 👋 ,

Working on a Fastify implementation as I see there is none at the moment.

Current stack consists of Fastify with Typescript for the code and SQLite3 + Knex for storage. At this stage it seems like the easiest for rapid prototyping bit I'm open to suggestions if speed becomes a bigger focal point.

**Comment by geromegrignon** [comment]
Hi, thanks for this new implementation!
